requirements-aprl.txt

```diff
@@ -1,2 +1,3 @@
+git+git://github.com/HumanCompatibleAI/ilqr.git#egg=ilqr
```

Comment:
What's going on here? This seems unrelated to the overall upgrade to 3.7

Reply:
It was previously in `requirements.txt`, but I realised it's not needed in `modelfree`. Didn't seem worth making a separate PR for it...

Reply:
Makes sense